### ISSUE TITLE
Showcase Django users on the homepage

### ISSUE THREAD

**Issue by thibaudcolas** [django/djangoproject.com#1698]
Follow-up to #110. On the homepage, we have [commented-out](https://github.com/django/djangoproject.com/blob/main/djangoproject/templates/homepage.html#L68) code showcasing Django users. What do people think of resurrecting this, to make Django more compelling by showcasing "big names" that are universally recognized?

Here’s what the current implementation looks like as-is:

![Django users showcase screenshot, titled Who is Django For?](https://github.com/user-attachments/assets/2741efa3-00f5-426d-bd25-60614c5e23fe)

A lot of other projects do this. For example here’s what [Ruby on Rails](https://rubyonrails.org/) does:

![Rails users showcase screenshot, titled You’re in Good Company](https://github.com/user-attachments/assets/46666326-12b9-4230-af50-7650ba72c695)

**Comment by marksweb** [comment]
I like this rails example. Looks less like a backend dev built it. 

I know we all love how easy bootstrap cards are, but a grid of icons works well for this purpose.

**Comment by arpan8925** [comment]
May I work on this Issue and Make the UI, Similar like Ruby Rails Brand Section ?

**Comment by ulgens** [comment]
@arpan8925 I don't think you need to ask for permission, you can just state you're going to handle it and submit a PR.

**Comment by arpan8925** [comment]
@ulgens  Thanks for mentioning, I am checking
